src/rpc/rawtransaction.cpp

```diff
@@ -258,19 +258,23 @@ UniValue verifytxoutproof(const JSONRPCRequest& request)
     if (request.fHelp || request.params.size() != 1)
         throw std::runtime_error(
             "verifytxoutproof \"proof\"\n"
-            "\nVerifies that a proof points to a transaction in a block, returning the transaction it commits to\n"
-            "and throwing an RPC error if the block is not in our best chain\n"
+            "\nVerifies that a proof points to a transaction in a block, returning the blockhash and transactions it commits to\n"
+            ", throwing an RPC error if the block is not in our best chain\n"
```

Comment:
Nit, `,` is the above line?

Reply:
I was gonna mention that but chose not to. Agree though, I think that would look better.